sdk/typescript/src/utils/format.ts

```diff
@@ -31,3 +31,26 @@ export function formatType(type: string) {
     }
     return type;
 }
+
+const ADDRESS_TRIM_MAX_LENGTH = 8;
+
+export function trimAddress(address: string): string {
```

Comment:
Did you add this because you thought it was useful or? It doesn't seem to be used

Just asking 😄